docs/developer/guide-for-sps.md

```diff
@@ -4,63 +4,50 @@ sidebar_position: 1
 
 # Service Providers
 
-This page contains information about connecting services to
-RCIAM in order to allow user
-login through RCIAM and to receive users' attributes. RCIAM is connected
-to a wide range of academic and social Identity Providers that users can choose
-from in order to access your service.
+This page contains information about connecting services to OpenAIRE Login in
+order to allow user login through OpenAIRE Login and to receive users'
+attributes. OpenAIRE Login is connected to a wide range of academic and social
+Identity Providers that users can choose from in order to access your service.
```

Comment:
This page contains information about using the OpenAIRE AAI for securing access
to your service. The OpenAIRE AAI supports both OpenID Connect (an extension to
OAuth 2.0) and SAML 2.0. So the first thing you need to decide is which of the
two you are going to use. If you want you can also choose to secure some with
OpenID Connect and others with SAML. To secure your services you are also going
to need a library for the protocol you’ve selected. Once your
service is connected to the OpenAIRE AAI, users will be able to choose from
a wide range of academic and social Identity Providers in order to access your service.

Reply:
Done